.github/quarkusbuilditemdoc.java

```diff
@@ -89,12 +91,17 @@ private String getJavaDoc(JavaDocCapable<?> source) {
         MutableMultimap<String, Pair<Path, JavaClassSource>> multimap = Multimaps.mutable.sortedSet
                 .with(Comparator.comparing(o -> o.getTwo().getName()));
         String line;
+        List<String> keywordList = Arrays.asList("extends BuildItem {", "extends SimpleBuildItem {",
+                "extends MultiBuildItem {", "extends EmptyBuildItem {");
         for (Path path : paths) {
             Files.walkFileTree(path, new SimpleFileVisitor<Path>() {
                 @Override
-                public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IOException {
-                    if (file.toString().endsWith("BuildItem.java")) {
-                        process(multimap, file);
+                public FileVisitResult visitFile(Path path, BasicFileAttributes attrs) throws IOException {
+                    String fileContent = Files.readString(path, StandardCharsets.ISO_8859_1);
+                    boolean match = keywordList.stream().anyMatch(fileContent::contains);
+                    //if (file.toString().endsWith("BuildItem.java")) {            //original version
+                    if (path.toString().endsWith("BuildItem.java") || match) {  // edited version to include more build items
```

Comment:
No need to comment that, it' s in the Git history already 😉 
```suggestion
                    if (path.toString().endsWith("BuildItem.java") || match) {
```

Reply:
Also I think we don't care anymore if the path ends with `BuildItem.java`. Knowing that it extends from those types may probably be enough